pkg/artifactory/resource/repository/remote/resource_artifactory_remote_repository_test.go

```diff
@@ -10,7 +10,6 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/go-resty/resty/v2"
```

Comment:
Is this building for you? There's a reference to `*resty.Request` at https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/resource/repository/remote/resource_artifactory_remote_repository_test.go#L971

Reply:
https://github.com/jfrog/terraform-provider-artifactory/actions/runs/11669346571/job/32491100933#step:13:946

Reply:
I put it back. It was auto removed in my ide